go/engine/context.go

```diff
@@ -21,6 +23,7 @@ type Context struct {
 	UpdateUI    libkb.UpdateUI
 
 	LoginContext libkb.LoginContext
+	NContext     context.Context
```

Comment:
I guess since that's the historical style...

But it's always bugged me a bit, and thought going forward we could move to more idiomatic style:  https://golang.org/doc/effective_go.html#Getters